src/structure_dump.cpp

```diff
@@ -354,6 +384,70 @@ void structure::set_chiP_from_file(h5file *file, const char *dataset, field_type
   }
 }
 
+void structure::load_chunk_layout(const char *filename, boundary_region &br,
+                                  std::vector<grid_volume> gvs) {
```

Comment:
It seems like this should be two methods:
```
void structure::load_chunk_layout(const char *filename, boundary_region &br);
void structure::load_chunk_layout(std::vector<grid_volume> gvs, boundary_region &br);
```
where the former calls the latter.

Reply:
Done.